app/modes/library/agent.py

```
@@ -178,48 +319,64 @@ def generate_book_content_background(app_context, book_id, user_id, user_backgro
                         logger.info(
-                            f"Generated plan for {topic.name}: {actual_chapters} chapters (estimate was 5, diff: {chapters_diff}). New total: {progress.total_chapters}"
+                            f"Generated plan for {topic.name}: {actual_chapters} chapters (estimate was 5, diff: {chapters_diff})."
                         )
 
-                        # Reload chapters
                         chapters = (
                             ChapterMode.query.filter_by(topic_id=topic.id)
                             .order_by(ChapterMode.step_index)
                             .all()
                         )
                     except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)